lldb-eval/eval_test.cc

```diff
@@ -94,6 +95,39 @@ class EvaluatorHelper {
     return ret;
   }
 
+  EvalResult EvalWithContext(
+      const std::string& expr,
+      const std::unordered_map<std::string, lldb::SBValue>& vars) {
+    EvalResult ret;
+
+    std::vector<lldb_eval::ContextVariable> ctx_vec;
+    ctx_vec.reserve(vars.size());
+    for (const auto& it : vars) {
+      ctx_vec.push_back({it.first.c_str(), it.second});
+    }
```

Comment:
```suggestion
    for (const auto& [name, value] : vars) {
      ctx_vec.emplace_back(name.c_str(), value);
    }
```

Reply:
Done, also implemented constructor for `ContextVariable`.